apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/share-view.test.tsx

```
@@ -0,0 +1,373 @@
+  cn: (...args: any[]) => args.filter(Boolean).join(" "),
+}));
+
+const mockTabs = [
+  { id: "email", label: "Email", icon: () => <div data-testid="email-tab-icon" /> },
+  { id: "webpage", label: "Web Page", icon: () => <div data-testid="webpage-tab-icon" /> },
+  { id: "link", label: "Link", icon: () => <div data-testid="link-tab-icon" /> },
+  { id: "app", label: "App", icon: () => <div data-testid="app-tab-icon" /> },
+];
+
+// Create proper mock survey objects
+const createMockSurvey = (type: "link" | "app", id: string = "survey1"): TSurvey => ({
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)